mico-core/src/main/java/io/github/ust/mico/core/exception/KafkaFaasConnectorInstanceAlreadyIncludedWithSameVersionInMicoApplicationException.java

```diff
@@ -0,0 +1,17 @@
+package io.github.ust.mico.core.exception;
+
+public class KafkaFaasConnectorInstanceAlreadyIncludedWithSameVersionInMicoApplicationException extends Exception {
```

Comment:
I do not think that we need this exception at all. The whole purpose of the current changes to mico is to support multiple KFconnectors with the same version in the same application!